base/applications/taskmgr/taskmgr.c

```diff
@@ -898,7 +903,7 @@ void TaskManager_OnEnterMenuLoop(HWND hWnd)
     SendMessageW(hStatusWnd, SB_SETTEXT, (WPARAM)0, (LPARAM)L"");
 }
 
-void TaskManager_OnExitMenuLoop(HWND hWnd)
+void TaskManager_EnableStatusbar(HWND hWnd)
```

Comment:
```suggestion
void TaskManager_EnableStatusBar(HWND hWnd)
```
and similar change for the Disable function above.